Resources/services.xml

```diff
@@ -6,8 +6,12 @@
         <service id="frosh_ion_cube_checker.plugin_loader" class="FroshIonCubeChecker\Components\PluginLoader">
             <argument type="service" id="models" />
         </service>
+        <service id="frosh_ion_cube_checker.plugin_path_service">
```

Comment:
Please test your code!
Fatal error: Uncaught Symfony\Component\DependencyInjection\Exception\RuntimeException: The definition for "frosh_ion_cube_checker.plugin_path_service" has no class. If you intend to inject this service dynamically at runtime, please mark it as synthetic=true. If this is an abstract definition solely used by child definitions, please add abstract=true, otherwise specify a class to get rid of this error. in /var/www/html/swcomposer/vendor/symfony/dependency-injection/Compiler/CheckDefinitionValidityPass.php:69 Stack trace: #0 /var/www/html/swcomposer/vendor/symfony/dependency-injection/Compiler/Compiler.php(141): Symfony\Component\DependencyInjection\Compiler\CheckDefinitionValidityPass->process(Object(Symfony\Component\DependencyInjection\ContainerBuilder)) #1 /var/www/html/swcomposer/vendor/symfony/dependency-injection/ContainerBuilder.php(788): Symfony\Component\DependencyInjection\Compiler\Compiler->compile(Object(Symfony\Component\DependencyInjection\ContainerBuilder)) #2 /var/www/html/swcomposer/vendor/shopware/shopw in /var/www/html/swcomposer/vendor/symfony/dependency-injection/Compiler/CheckDefinitionValidityPass.php on line 69

Reply:
@shyim  I'm very sorry. Forgot to push my latest changes. Just updated the code and retested to be sure. Hope it works now.